docs/data-layer.md

```diff
@@ -0,0 +1,178 @@
+# Data Layer
+
+## Redux
+
+In eHoldings, data is connected on `routes` only. This is done via
+`react-redux`'s `connect` decorator.
+
+``` javascript
+connect(({ eholdings: { data } }) => ({
+  // ...mapStateToProps
+  // data needed by the component is accessed here
+}), {
+  // ...mapDispatchToProps
+  // actions used to send requests for data are defined here
+});
+```
+
+When data is needed, you must first request it with one of the actions
+defined in `mapDispatchToProps`. The data received in
+`mapStateToProps` will then update with a pending request, and update
+again once the request resolves or rejects.
+
+## Models
+
+In eHoldings, you must define a model to request and store data in the
```

Comment:
`In eHoldings` is echoed and also probably provides context the reader already has given where they're reading this